src/theme/components/hero.ts

```diff
@@ -220,12 +314,42 @@ const campaign = definePartsStyle({
     maxWidth: { md: "1248px" },
     position: { md: "relative" },
     zIndex: 2,
+    a: {
+      color: getLinkColor("default", "light", foregroundColor, isDarkText),
+      _hover: {
+        color: getLinkColor("hover", "light", foregroundColor, isDarkText),
+      },
+      _visited: {
+        color: getLinkColor("visited", "light", foregroundColor, isDarkText),
+        svg: {
+          fill: getLinkColor("visited", "light", foregroundColor, isDarkText),
+        },
+      },
+    },
+    ".chakra-heading": {
```

Comment:
Is this done because it's hard to override or add the value directly to the component?

Reply:
Using the class name was the easiest way to target and override the Heading styles.